src/Symfony/Component/Notifier/Bridge/GoogleChat/Tests/GoogleChatTransportTest.php

```diff
@@ -31,14 +31,15 @@ final class GoogleChatTransportTest extends TransportTestCase
     /**
      * @return GoogleChatTransport
      */
-    public function createTransport(?HttpClientInterface $client = null): TransportInterface
+    public function createTransport(?HttpClientInterface $client = null, string $threadKey = null): TransportInterface
     {
-        return new GoogleChatTransport('My-Space', 'theAccessKey', 'theAccessToken=', $client ?: $this->createMock(HttpClientInterface::class));
+        return new GoogleChatTransport('My-Space', 'theAccessKey', 'theAccessToken=', $threadKey, $client ?: $this->createMock(HttpClientInterface::class));
     }
 
     public function toStringProvider(): iterable
     {
         yield ['googlechat://chat.googleapis.com/My-Space', $this->createTransport()];
+        yield ['googlechat://chat.googleapis.com/My-Space?thread_key=abcdefg', $this->createTransport(null, 'abcdefg')];
```

Comment:
same

Reply:
Nope, this is our DSN format where we want to use snake_case, like all other bridges.